.travis.yml

```diff
@@ -54,3 +54,5 @@ notifications:
 cache:
   apt: true
   pip: true
+  directories:
```

Comment:
Please remove this change - this change shouldn't be addressed in this pull request.